qa/integration-tests/src/test/java/io/camunda/zeebe/it/health/DiskSpaceRecoveryIT.java

```diff
@@ -48,7 +48,7 @@ final class DiskSpaceRecoveryIT {
           .withEnv("ZEEBE_BROKER_EXPERIMENTAL_RAFT_PREFERSNAPSHOTREPLICATIONTHRESHOLD", "0")
           .withEnv("ZEEBE_BROKER_DATA_LOGSEGMENTSIZE", "1MB")
           .withEnv("ZEEBE_BROKER_NETWORK_MAXMESSAGESIZE", "1MB")
-          .withEnv("ZEEBE_BROKER_DATA_DISK_FREESPACE_PROCESSING", "10MB")
+          .withEnv("ZEEBE_BROKER_DATA_DISK_FREESPACE_PROCESSING", "8MB")
```

Comment:
:question: Why was this required?

Reply:
The test times out otherwise. I can't really wrap my head around it (for example: how much disk space is from segments, how much from RocksDB) and it's a but unsatisfying to just change a magic number but currently I don't see a better option.

The only difference here is that we'll always have at least two segments (each 1MiB in this test) instead of just one.